src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -42,6 +59,26 @@ const getVipHelperSuffix = (
   return '';
 };
 
+const getVipValidationsById = (
+  validationsInfoString?: Cluster['validationsInfo'],
+): { [key: string]: string | undefined } => {
+  const validationsInfo = stringToJSON<ValidationsInfo>(validationsInfoString) || {
+    hostsData: [],
+    network: [],
+    configuration: [],
+  };
+  const failedDhcpAllocationMessageStub = 'VIP IP allocation from DHCP server has been timed out';
+  return validationsInfo.network.reduce((lookup, validation) => {
+    if (['api-vip-defined', 'ingress-vip-defined'].includes(validation.id)) {
+      lookup[validation.id] =
+        validation.status === 'failure' && validation.message.match(failedDhcpAllocationMessageStub)
+          ? validation.message
+          : undefined;
```

Comment:
Considering previous comment, this will probably need to keep `lookup[validation.id]` which is either `undefined` or an already found message.

Reply:
I still think we will reverse this search. At latest when fixing bug which will be opened right after adding another error message on backend. :-) Can be changed in a follow-up.